README.md

```diff
@@ -59,34 +60,29 @@ class Shot
   attribute :id, Integer
   attribute :name, String
   attribute :url, String
-  attribute :image_url, String
+  attribute :image_url, String, as: :image # You can define custom attribute mapping
```

Comment:
@brennovich I think in this case `alias` would be a more concise name for that option.
The name `as` might have an ambiguous meaning. SimpleForm and Formtastic, for example, uses `as` to determine the field type:
```ruby
f.input :age, as: :integer
```

Got it?